apps/frontend/src/components/UserFeedbackModal/RatingForm.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import styles from './UserFeedbackModal.module.scss';
+import { ClassData } from './types';
+
+interface RatingFormProps {
+  currentClass: ClassData;
+}
+
+export function RatingsForm({ currentClass }: RatingFormProps) {
+  const [ratings, setRatings] = useState({
+    usefulness: 0,
+    difficulty: 0,
+    workload: 0
+  });
+
+  const handleRatingClick = (type: keyof typeof ratings, value: number) => {
+    setRatings(prev => ({
+      ...prev,
+      [type]: prev[type] === value ? 0 : value
+    }));
+  };
+
+  const renderRatingScale = (
```

Comment:
I would recommend splitting this into a separate component within this file rather than using a render function, as I personally find render functions a bit less structured and readable.